guardrails/utils/hub_telemetry_utils.py

```diff
@@ -118,9 +118,14 @@ def create_new_span(
             span_name,  # type: ignore (Fails in Python 3.8 for invalid reason)
             context=self.extract_current_context() if has_parent else None,
         ) as span:
-            if is_parent:
-                # Inject the current context
-                self.inject_current_context()
-
-            for attribute in attributes:
-                span.set_attribute(attribute[0], attribute[1])
+            try:
+                if is_parent:
+                    # Inject the current context
+                    self.inject_current_context()
+                for attribute in attributes:
+                    span.set_attribute(attribute[0], attribute[1])
+
+                span.set_status(trace.Status(trace.StatusCode.OK))
+            except Exception as e:
+                logging.error("An error occured during sopan creation: " + str(e))
+                span.set_status(trace.Status(trace.StatusCode.ERROR))
```

Comment:
This is in Hub telemetry utils - the anonymized data collected by Guardrails when the enviornment is configured to send data over. Please see telemetry_utils.py for the telemetry taht's not collected by guardrails